### PR TITLE
Add some terms to lint file

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

- Add a few terms to the lint file. Goal: avoid typos in product names or abbreviations in future commits.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - [x]  I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: .textlintrc.json [hunk 1]
```diff
@@ -126,6 +127,8 @@
         "Home Assistant Supervised",
         "Home Assistant Supervisor",
         "Home Assistant",
+        "Home Assistant SkyConnect",
```
Comment (frenck) [inline .textlintrc.json:130]: Maybe add `SkyConnect` as a single word too?

**Review by frenck — CHANGES_REQUESTED** [review]
Looks like it found issues that need to be addressed as well in that case:

image

File: .textlintrc.json (hunk 1 above)
Comment (c0ffeeca7) [reply .textlintrc.json:130]: Done. Thank you for the hint!

**Comment by c0ffeeca7** [comment]
> Looks like it found issues that need to be addressed as well in that case:
> 
> image

I removed the term from the linter because I realized it is used in lowercase in path names. Or is there a way to allow for spelling variants in the lint file?

**Comment by frenck** [comment]
Interesting, case of zigbee2mqtt, the linter should not have triggered on that.
Seems like it was a known issue before: <https://github.com/sapegin/textlint-rule-terminology/issues/38>

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**